subsys/usb/class/hid/core.c

```diff
@@ -211,7 +211,8 @@ static struct usb_ep_cfg_data hid_ep_data[] = {
 
 static void hid_interface_config(u8_t bInterfaceNumber)
 {
-	hid_cfg.if0.bInterfaceNumber = bInterfaceNumber;
+	UNALIGNED_PUT(bInterfaceNumber,
+		      &hid_cfg.if0.bInterfaceNumber);
 }
```

Comment:
It does not need alignment.

Reply:
You might be right, I have no knowledge about how Intel S1000 supports unaligned accesses